cobra/cmd/init.go

```diff
@@ -210,7 +210,7 @@ func initConfig() {
 
 	// If a config file is found, read it in.
 	if err := viper.ReadInConfig(); err == nil {
-		fmt.Println("Using config file:", viper.ConfigFileUsed())
+		fmt.Fprintln(os.Stderr, "Using config file:", viper.ConfigFileUsed())
```

Comment:
Why is this stderr?

Reply:
That's because of #774:

> This has been a minor annoyance for me with the Cobra codegen tool. Error messages and debugging information ("Using config file:") should be printed to stderr and not stdout.

Reply:
that's informational, not an error though...

Reply:
I think that one thing is what's preventing me from merging. I don't like those 2 lines.

Reply:
I think that it belong to stderr too, because that info does not belong to the logic of the app/command. Instead, it belong to the *middleware/library* that is handling configs/flags/options.

Nevertheless, since the main point of this PR is to merge trivial non-opinionated patches, I will just remove it from the set. Then, #774 can be handled independently.

Reply:
@eparis, it is removed now.